### ISSUE TITLE
Regression: Rebuilds only once in v4.0.0. Works in v3.2.2

### ISSUE THREAD

**Issue by btakita** [rollup/rollup-watch#51]
Upon upgrading to v4.0.0, my watch scripts only fire on the first change.

Downgrading to v3.2.2 fixes the issue. I don't know the cause of the issue & I will further investigate if nobody else has already done so.

useChokidar is true in this case.

**Comment by btakita** [comment]
Adding the rollup option `{watch: {useChokidar: false}}` fixes the issue

**Comment by Rich-Harris** [comment]
I suspect this is because chokidar didn't get used at all prior to version 4, due to a silly bug. Are you able to create a repo reproducing the issue, so we can investigate further? Thanks

**Comment by btakita** [comment]
This issue is related to: https://github.com/paulmillr/chokidar/issues/556

**Comment by btakita** [comment]
I'm trying to pass in the config:

`
watch: {
  chokidar: {usePolling: true}
}
`

However, `inputOptions` from function `mergeOptions` in `bin/rollup` does not include the `watch` member.

**Comment by Rich-Harris** [comment]
This repo is deprecated — `rollup.watch` is now part of core. If this is still an issue with the latest Rollup version, could you open a ticket over there please? thanks
